keyboards/xcstore/cc67/matrix.c

```diff
@@ -0,0 +1,196 @@
+/* Copyright 2022 XCStore
+ * Copyright 2022 Yulei (Astro) <https://github.com/yulei>
+ * Copyright 2022 HorrorTroll <https://github.com/HorrorTroll>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <string.h>
+#include <hal.h>
+#include "quantum.h"
+#include "timer.h"
+#include "wait.h"
+#include "print.h"
+#include "matrix.h"
+
+/**
+ *
+ * Row pins are input with internal pull-down.
+ * Column pins are output and strobe with high.
+ * Key is high or 1 when it turns on.
+ *
+ */
+/* matrix state(1:on, 0:off) */
+static matrix_row_t matrix[MATRIX_ROWS];
+static matrix_row_t matrix_debouncing[MATRIX_COLS];
+static bool debouncing = false;
+static uint16_t debouncing_time = 0;
+
+// user-defined overridable functions
+
+__attribute__((weak)) void matrix_init_kb(void) { matrix_init_user(); }
+
+__attribute__((weak)) void matrix_scan_kb(void) { matrix_scan_user(); }
+
+__attribute__((weak)) void matrix_init_user(void) {}
+
+__attribute__((weak)) void matrix_scan_user(void) {}
+
+// helper functions
+void matrix_init(void)
+{
+    //debug_enable = true;
+    palSetLineMode(LINE_PB12,         PAL_MODE_OUTPUT_PUSHPULL);
```

Comment:
This should be refactored to 
* use `lite`
* use QMK standard GPIO defines

Reply:
done, just refactor to use lite instead full